src/components/App.css

```diff
@@ -0,0 +1,22 @@
+
+.app {
+  background-color: #0E0E0E;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  letter-spacing: 2px;
```

Comment:
Why letter spacing?

Reply:
We thought it looked a bit squashed - happy to delete it and we can decide together at a later point